mimesis/providers/path.py

```diff
@@ -9,10 +8,10 @@
 class Path(BaseProvider):
     """Class that provides methods and property for generate paths."""
 
-    def __init__(self, *args, **kwargs):
-        # TODO: platform should be a parameter
+    def __init__(self, platform=sys.platform, *args, **kwargs):
```

Comment:
Maybe it is a good idea to create a structure with: 
1. name
2. path separator
3. other useful things

@lk-geimfari ?